src/app/components/harvest/pages/list/list.component.spec.ts

```diff
@@ -118,6 +119,8 @@ describe("ListComponent", () => {
     )?.nativeElement as T;
   }
 
+  beforeAll(() => testRunnersDefaultTimezone = Settings.defaultZone);
+
```

Comment:
Ok, this mocking behavior is repeated in several files. I wonder if we can't do better?

```javascript
// Helpers.ts

const withDefaultZone = function (zone, f) {
  const fullName = `With defaultZone: ${zone}`;
  describe(fullName, () => {
     beforeEach(() => {
         Settings.defaultZone = zone;
     });

    afterEach(() => {
        Settings.defaultZone = null;
    });
    
    f.call(this);
  });
};

// usage

Helpers.withDefaultZone(FixedOffsetZone.utcInstance, () => {

   it('blah blah blah' , () => { ... } );

});
```